### ISSUE TITLE
Generating image using Labels

### ISSUE THREAD

**Issue by rut00** [yuval-alaluf/stylegan3-editing#35]
I have the StyleGAN3 network trained on the labeled dataset.
I want to know how and where to pass a label to generate images while editing?
My labels are in numeric form like 1 and 2.

**Comment by rut00** [comment]
For generating images with label, we added below lines in `edit_synthetic.py` file under the function `get_random_image`:

```
label = torch.zeros([1, G.c_dim], device=device)
# Label 1
label[:, 1] = 1

label = torch.zeros([1, G.c_dim], device=device)
# Label 2
label[:, 2] = 1
```

Thank you for your help.
